### ISSUE TITLE
Early stopping

### ISSUE THREAD

**Issue by ghost** [bing0037/keras-yolo3#1]
In training raccoon data, having the error:

Epoch 00094: ReduceLROnPlateau reducing learning rate to 9.999999939225292e-10.
Epoch 00094: early stopping.

how to solve the problem?

**Comment by 2017TJM** [comment]
144/144 [==============================] - 47s 324ms/step - loss: 73.5387 - val_loss: 154.7555
Epoch 00079: early stopping
